File: ui/src/component/Item.js
import React, {Component} from 'react';

export default class Item extends Component{
    render(){
        return(
            <div className="row m-1" style = {{height: 100, overflow:"auto"}} >
                <div className = 'col border text-left'>
                        <h5>Item {this.props.name}</h5>
                        <h6>Dexcription {this.props.description}</h6>
                        <small>Price : {this.props.price}</small>
                </div>        
            </div>
        );
    }
}

File: app/module/Item.java
package module;

public class Item {
    private String idMerch;
    private String idItem;
    private String name;
    private int price;
    private String description;

    public Item(String name, int price, String description){
        this.name = name;
        this.price = price;
        this.description = description;
    }
}
